example/ssd/dataset/iterator.py

```diff
@@ -159,21 +161,22 @@ def _data_augmentation(self, data, label):
                 xmax = int(crop[2] * width)
                 ymax = int(crop[3] * height)
                 if xmin >= 0 and ymin >= 0 and xmax <= width and ymax <= height:
-                    data = data[ymin:ymax, xmin:xmax, :]
+                    data = mx.img.fixed_crop(data, xmin, ymin, xmax-xmin, ymax-ymin)
                 else:
                     # padding mode
                     new_width = xmax - xmin
                     new_height = ymax - ymin
                     offset_x = 0 - xmin
                     offset_y = 0 - ymin
-                    data_bak = data
+                    #TODO need ndarray to support this
+                    data_bak = data.asnumpy()
                     data = np.full((new_height, new_width, 3), 128.)
                     data[offset_y:offset_y+height, offset_x:offset_x + width, :] = data_bak
+                    data = mx.nd.array(data)
```

Comment:
nd.Pad only suports 4D and 5D and in CWH format which is different than a image. too much work here and leave it later.